recipes-asteroid/asteroid-launcher/asteroid-launcher_git.bb

```diff
@@ -23,4 +29,14 @@ do_install_append() {
     install -d ${D}/usr/lib/systemd/user/default.target.wants/
     cp ../asteroid-launcher.service ${D}/usr/lib/systemd/user/
     ln -s ../asteroid-launcher.service ${D}/usr/lib/systemd/user/default.target.wants/asteroid-launcher.service
+
+    install -d ${D}/etc/systemd/system/user@.service.d/
+    cp ../localeEnv.conf ${D}/etc/systemd/system/user@.service.d/locale.conf
+
+    install -d ${D}/var/lib/environment/ceres/
+    cp ../locale.conf ${D}/var/lib/environment/ceres/locale.conf
+    chown 1000:1000 ${D}/var/lib/environment/ceres/locale.conf # ceres:ceres
```

Comment:
This causes the QA warnings, but because the user ceres does not exist at this point, running `chown ceres:ceres` is not possible.